devimint/src/external.rs

```diff
@@ -33,6 +34,26 @@ pub struct Bitcoind {
 impl Bitcoind {
     pub async fn new(processmgr: &ProcessManager) -> Result<Self> {
         let btc_dir = utf8(&processmgr.globals.FM_BTC_DIR);
+        let conf = format!(
+            "\
+regtest=1
```

Comment:
`r#" ... "#';` raw strings might work better, possibly move to another file and `include_str!(...);` it anyway.

Reply:
surprising that include_str! works with format!